src/main/java/org/embulk/input/gcs/GcsFileInput.java

```diff
@@ -91,28 +91,50 @@ static FileList listFiles(final PluginTask task) {
 
     // String nextToken = base64Encode(0x0a + ASCII character according to utf8EncodeLength position+ filePath);
     static String base64Encode(final String path) {
+        byte[] lengthVarint;
         byte[] encoding;
         byte[] utf8 = path.getBytes(StandardCharsets.UTF_8);
         LOG.debug("path string: {} ,path length:{} \" + ", path, utf8.length);
 
         int utf8EncodeLength = utf8.length;
-        if (utf8EncodeLength >= 128) {
+        if (utf8EncodeLength >= 65_535) {
```

Comment:
It's understandable that this `base64Encode` method can handle 65536 bytes at most by itself (regardless of the limitation for `task.getLastPath()`), but it is a little bit confusing for code readers that it has a different limitation from `task.getLastPath()`.

Can you align this with `task.getLastPath()` and leave a comment to explain why limited by that number?

Reply:
Aligned byte limit to 1024 bytes and added a comment.
https://github.com/embulk/embulk-input-gcs/pull/72/commits/db7e7b97aa050328abcf7e8f785be2f5b52a6cdc